generated-platform-project/quarkus-cxf/integration-tests/quarkus-cxf-integration-test-mtom/pom.xml

```diff
@@ -10,6 +10,9 @@
   </parent>
   <artifactId>quarkus-cxf-integration-test-mtom</artifactId>
   <name>Quarkus Platform - CXF - Integration Tests - quarkus-cxf-integration-test-mtom</name>
+  <properties>
+    <maven.test.skip>true</maven.test.skip>
```

Comment:
mtom regression ? @vkasala fyi

Reply:
Not really a regression in MTOM, it is rather that the test has changed recently and became longer-running which makes it more likely to fail on GHA infra - see https://github.com/quarkiverse/quarkus-cxf/issues/1067
In the main platform branch, we shortened the default duration time already and the test is not disabled anymore.